metals/src/main/resources/db/migration/V1__Create_tables.sql

```diff
@@ -8,7 +8,7 @@ create table dependency_source(
 
 -- The relationship between what library dependency sources under .metals/readonly/**
 -- map to which build targets.
-create table sbt_digest(
+create table digest(
```

Comment:
Shouldn't there be a new script altering table name?

Reply:
I don't think we have a way to do it, but with a new name it shouldn't cause any issues I think. @olafurpg ?

Reply:
I do not know what mechanism is used underneath, but migration scripts in my experience should almost never change. It may cause issues for clients who have already have them applied (different script checksum etc.)

Anyway, let me try to understand how this part actually works

Reply:
Yup, I believe you should create a file `migration/V4_rename_digest.sql` with:
```sql
alter table sbt_digest rename to digest;
```
(for reference https://www.h2database.com/html/commands.html#alter_table_rename)

Reply:
Thanks, I will do the rename in a separate commit - it's not strictly related to maven